framework/src/security/security_api.c

```diff
@@ -0,0 +1,30 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <pthread.h>
+#include <security/security_api.h>
+#include <tinyara/security_hal.h>
+#include <tinyara/seclink.h>
+
+#include "security_utils.h"
+#include "security_common.h"
+
```

Comment:
Is this file missing API definitions? Please ignore the comment if not relevant.

Reply:
API definitions are separated. so it can be ignored.